src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509ChainPolicy.cs

```diff
@@ -68,6 +72,20 @@ public X509VerificationFlags VerificationFlags
             }
         }
 
+        public X509ChainTrustMode TrustMode
+        {
+            get
+            {
+                return _trustMode;
+            }
+            set
+            {
+                if (value < X509ChainTrustMode.System || value > X509ChainTrustMode.CustomRootTrust)
+                    throw new ArgumentException(SR.Format(SR.Arg_EnumIllegalVal, nameof(value)));
```

Comment:
I don't think there are tests for this exception.  Please add a test where it assigns `(X509ChainTrustMode)-1` and `(X509ChainTrustMode)2`.  (Probably ChainTests.cs, seems like a reasonable Theory (InlineData(-1), InlineData(2)))